frontend/pages/donations/make.tsx

```diff
@@ -1,15 +1,30 @@
 import Layout from "@/components/Layout";
+import { Button } from "@material-ui/core";
 import Container from "@material-ui/core/Container";
+import { makeStyles } from "@material-ui/core/styles";
 import DropIn from "braintree-web-drop-in-react";
 import { GetServerSideProps } from "next";
+import Link from "next/link";
 import { useRouter } from "next/router";
 import { FC, useEffect, useState } from "react";
 import axiosWithoutAuth from "../../axiosWithoutAuth";
+
+const useStyles = makeStyles({
+  root: {
+    height: "100%",
+    display: "flex",
+    alignItems: "center",
+    justifyContent: "center",
+    marginTop: "1.5rem",
+    marginBottom: "2rem",
+  },
+});
 interface DonateProps {
   clientToken: string;
 }
 
 const Donate: FC<DonateProps> = (props: DonateProps) => {
+  const classes = useStyles();
   const [clientToken, setClientToken] = useState(props.clientToken);
```

Comment:
@sunzae99, maybe we should use state for `success` and `error`, to help with conditional rendering.

We could do it like this:
```
const [success, setSuccess] = useState(false); 
const [error, setError] = useState(false); 
```